packages/mesos-modules/build

```diff
@@ -1,15 +1,29 @@
 #!/bin/bash
+set -o nounset -o pipefail
+
 pushd /pkg/src/mesos-modules/
 ./bootstrap
 popd
 
-source /opt/mesosphere/environment.export
+export LD_LIBRARY_PATH=/opt/mesosphere/lib
+
+# Add `protoc` to the path.
+export PATH=/opt/mesosphere/lib/mesos/3rdparty/bin:$PATH
```

Comment:
why can't we move / copy / symlink into `$PKG_PATH/bin` in the mesos package if this is critical (or just everything in the 3rdparty/bin folder)?